src/components/Modals/Send/views/Details.tsx

```diff
@@ -44,7 +44,7 @@ import { SendMaxButton } from '../SendMaxButton/SendMaxButton'
 const MAX_COSMOS_SDK_MEMO_LENGTH = 256
 
 export const Details = () => {
-  const { control, setValue } = useFormContext<SendInput>()
+  const { control, setValue, trigger } = useFormContext<SendInput>()
```

Comment:
validation safety, might work without because of how `handleInputChange` works, but a good practice to trigger fields when we set their value programmatically from outside the component, as we do here with QR code amount setting